### ISSUE TITLE
Open terminal on keypress

### ISSUE THREAD

**Issue by apachano** [GentenStudios/Phoenix#70]
## Story
What are we solving? What does this look like?

## MVP
What are the minimum features needed for this story?
-  [ ] When pressing a key (return as default) while the cursor is focused on the game, the terminal focuses

**Comment by tobyplowy** [comment]
@apachano This seems done yes?

**Comment by apachano** [comment]
Nope, this does not currently happen. You must press escape (same as accessing the debug tools) then click into the terminal.

**Comment by tobyplowy** [comment]
@beeperdeeper089 Seeing as you are working on the terminal can you consider resolving this?
